packages/networks/src/networks/pulsechain.ts

```diff
@@ -0,0 +1,49 @@
+import { NetworkConfig } from '@unlock-protocol/types'
+
+export const pulsechain: NetworkConfig = {
+    chain: 'pulsechain',
+    description: 'pulsechain mainnet.',
+    explorer: {
+        name: 'Pulsechain',
+        urls: {
+            address: (address: string) =>
+                `https://scan.pulsechain.com/address/${address}`,
+            base: `https://scan.pulsechain.com/`,
+            token: (address: string, holder: string) =>
+                `https://scan.pulsechain.com/token/${address}?a=${holder}`,
+            transaction: (hash: string) => `https://scan.pulsechain.com/tx/${hash}`,
+        },
+    },
+    featured: true,
+    fullySubsidizedGas: false,
+    id: 369,
+    isTestNetwork: false,
+    keyManagerAddress: '',
+    maxFreeClaimCost: 10000,

```

Comment:
```suggestion
    maxFreeClaimCost: 0,
```

We won't subsidize gas on this chain for now.